src/mapwindow.jl

```diff
@@ -29,6 +29,13 @@ AbstractUnitRanges, in which case the specified ranges are used for
 `border` specifies how the edges of `img` should be handled; see
 `imfilter` for details.
 
+Finally `imginds` allows to omit unnecessary computations, if you want to do things
+like `mapwindow` on a subimage, or a strided variant of mapwindow.
+It works as follows:
+```julia
+mapwindow(f, img, window, border, (2:5, 1:2:7)) == mapwindow(f,img,window,border)[2:5, 1:2:7]
+```
```

Comment:
Perhaps add "except more efficiently because it omits computation of the unused values."